src/histolab/tiler.py

```diff
@@ -159,7 +134,7 @@ def level(self) -> int:
     @level.setter
     def level(self, level_: int):
```

Comment:
do not mix private and public methods/properties. In a class public methods should go first then the privates and in each section (priv and pub) they should be ordered alphabetically